themes/SuiteP/css/suitep-base/editview.scss

```diff
@@ -32,7 +32,6 @@ div#searchResults {
 }
 
 #EditView #reminders input[type="button"]#reminder_add_btn {
```

Comment:
it's empty

Reply:
I have when through all my changes to look for the same issue, and I have removed the empty ones.